artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -0,0 +1,1513 @@
+                "401":
+                  $ref: "#/components/responses/Generic401"
+                "403":
+                  $ref: "#/components/responses/Generic403"
+                "500":
+                  $ref: "#/components/responses/Generic500"
+                "503":
+                  $ref: "#/components/responses/Generic503"
+              security:
+                - { }
+
+      responses:
+        "201":
+          description: Created
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Subscription'
+        "202":
+          description: Request accepted to be processed. It applies for async creation process.
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SubscriptionAsync'
+        "400":
+          $ref: '#/components/responses/Generic400'
+        "401":
+          $ref: '#/components/responses/Generic401'
+        "403":
+          $ref: '#/components/responses/Generic403'
+        "409":
+          $ref: '#/components/responses/Generic409'
+        "500":
+          $ref: '#/components/responses/Generic500'
+        "503":
+          $ref: '#/components/responses/Generic503'
+    get:
+      tags:
+        - event-type notification subscription
+      summary: 'Retrieve a list of event-type event subscription'
+      description: Retrieve a list of event-type event subscription(s)
+      operationId: retrieveSubscriptionList
+      parameters:
+        - $ref: '#/components/parameters/x-correlator'
+      security:
+        - openId:
+          - event-type:subscriptions:read
+      responses:
+        "200":
+          description: List of event subscription details
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Subscription'
+        "400":
+          $ref: '#/components/responses/Generic400'
+        "401":
+          $ref: '#/components/responses/Generic401'
+        "403":
+          $ref: '#/components/responses/Generic403'
+        "500":
+          $ref: '#/components/responses/Generic500'
+        "503":
+          $ref: '#/components/responses/Generic503'
+  /subscriptions/{subscriptionId}:
+    get:
+      tags:
+        - event-type notification subscription
+      summary: 'Retrieve a event-type event subscription'
+      description: retrieve event subscription information for a given subscription.
+      operationId: retrieveSubscription
+      security:
+        - openId:
+          - event-type:subscriptions:read
```

Comment:
@bigludo7  As discussed here https://github.com/camaraproject/Commonalities/issues/163#issuecomment-2009923728, shouldn't we simply use

<api-name>:subscriptions:read
<api-name>:subscriptions:delete